yt_dlp/extractor/pornhub.py

```diff
@@ -269,6 +269,8 @@ def _real_extract(self, url):
         self._login(host)
 
         self._set_cookie(host, 'age_verified', '1')
+        self._set_cookie(host, 'accessAgeDisclaimerPH', '1')
+        self._set_cookie('thumbzilla.com', 'accessAgeDisclaimerTZ', '1')
```

Comment:
Why do you set a cookie for a third party site? This shouldn't be required.

Reply:
the extractor is used for both sites

```py
class PornHubIE(PornHubBaseIE):
    IE_DESC = 'PornHub and Thumbzilla'
```

Reply:
Oh, didn't saw that. I'm sorry.

Reply:
after looking more closely at the code, I think you may be right that the thumbzilla cookie is unnecessary. The extractor seems to match thumbzilla URLs only. Everything is extracted from PH